src/db/models/index.ts

```diff
@@ -30,7 +30,7 @@ class SequelizeFactory implements SequelizeFactory {
                 transactionType: this.config.transactionType,
                 retry: {max: this.config.retry.max},
                 logQueryParameters: true,
-                logging: log.debug.bind(log)
+                logging: config.get('db.logging') ? log.debug.bind(log) : false
```

Comment:
I've added this, bc now db floods console with debug messages. Default behavior is not changed, but now you can override it by local config file

Reply:
This makes sense, but please be more specific introducing new config options. This particular one only controls the database logging, right? Please rename it appropriately. And also set it to `false` in `production.yaml`.